doc/modules/clustering.rst

```diff
@@ -278,6 +285,26 @@ small, as shown in the example and cited reference.
    D. Sculley, *Proceedings of the 19th international conference on World
    wide web* (2010)
 
+.. _k_medoids:
+
+K-Medoids
+=========
+
+:class:`KMedoids` is similar to :class:`KMeans` with some exceptions. First, instead of using the mean of each cluster as the cluster center, :class:`KMedoids` uses that data point as the cluster center (also known as the medoid) whose sum-of-distances to all cluster members is the smallest. Second, the distance metric used by :class:`KMedoids` can be freely chosen, whereas with :class:`KMeans` it is fixed to Euclidean distance.
+
+.. topic:: Examples:
+
+ * :ref:`example_cluster_plot_kmedoids_digits.py`: Applying K-Medoids on digits with various distance metrics.
+
+**Algorithm description:**
```

Comment:
As far as the k-medians difference is concerned - I've added the example from wikipedia: (0, 1), (1, 0), (2, 2). It's a very illustrative one, I think it fits well into the description. Should I add wikipedia to the references section?